tests/integration/test_vulnerability_detector/test_windows/test_cpe_indexing.py

```diff
@@ -130,22 +130,22 @@ def get_configuration(request):
 
```

Comment:
Maybe, it would be a good idea to add some logic to the definition of `system_data` due to it is too long with repetitive information.
Also, the standard of the repository is to use single quotes in case of literal string variables, for example:
```
"os_release": "1000"
```
Should be:
```
'os_release': '1000'
```